### PR TITLE
Update Pharas.yaml

### PR DESCRIPTION
**Pull request by Pharas**
Added my lit review 1 to the yaml file

### CODE REVIEW

File: scripts/people/Pharas.yaml [hunk 1]
```diff
@@ -6,3 +6,4 @@
   irc: Pharas
   name: Brian Escriche
   rit_dce: bke2759
+  litreview1:https://www.blogger.com/blogger.g?blogID=1052590409801248183#editor/target=post;postID=6719244841524002849;onPublishedMenu=allposts;onClosedMenu=allposts;postNum=0;src=postname
```
Comment (decause) [inline scripts/people/Pharas.yaml]: @Pharas just add a space after the ":" and I'll merge.

Comment (decause) [inline scripts/people/Pharas.yaml]: Also, that URL looks _kinda_ shady...

**Comment by Pharas** [comment]
Fixed the yaml and changed the link to be "less sketchy"

**Comment by decause** [comment]
@Pharas You did indeed make it look less sketchy on the _outside_ but under the hood, the link is still one that only you can see (some kind of blogger admin dashboard-y link)

Try finding the permalink by visiting your blog in a new tab, and use that one instead.

**Comment by Pharas** [comment]
Maybe now?

**Comment by decause** [comment]
:100:

**Pull request merged**